terraform/workspace_variables/dev.tfvars.json

```diff
@@ -0,0 +1,13 @@
+{
+  "environment_name": "dev",
+  "key_vault_name": "s165d01-kv",
+  "resource_group_name": "s165d01-rg",
+  "apply_qts_app_name": "apply-qts-dev",
+  "logging_service_name": "apply-qts-logit-ssl-drain-dev",
+  "paas_space": "tra-dev",
+  "postgres_database_name": "apply-qts-dev-pg-svc",
+  "redis_name": "apply-qts-dev-redis-svc",
+  "hostnames": ["dev-apply-qts"],
```

Comment:
Is this agreed with business?

Reply:
I'm going to get a decision on this, thanks for pointing it out!

Reply:
This probably is not required for dev environment @prabhakarreddy1234. We can ignore custom domain bit for now.